### ISSUE TITLE
Requests ELF file instead of copy/paste stack trace

### ISSUE THREAD

**Issue by GadgetRho** [me-no-dev/EspExceptionDecoder#26]
The readme file states that one should copy and paste the stack trace, but this tool wants an elf file.

**Comment by Torben80** [comment]
...Compile the sketch and it will not ask for elf...
someone wrote a few days ago

**Comment by MrUrdam** [comment]
This solved the problem for me.

**Comment by dresende** [comment]
Does not always work. You can find the elf in `Libraries/Arduino15/packages/esp8266/hardware/esp8266/**version**/bootloaders/eboot/eboot.elf`.

**Comment by pauloeduardosp** [comment]
i found the elf file, but i didn't know where i need to put then

**Comment by dresende** [comment]
The elf file is needed when the ESP Exception Decoder window opens and requests it. If it just works you don't need to point to the elf file.

**Comment by zuqualla** [comment]
> 
> 
> Does not always work. You can find the elf in `Libraries/Arduino15/packages/esp8266/hardware/esp8266/**version**/bootloaders/eboot/eboot.elf`.

This file is also good for esp32?

**Comment by thataintme** [comment]
> ...Compile the sketch and it will not ask for elf...
> someone wrote a few days ago

Can you please tell me how to compile a .jar file? is that what a sketch is?

**Comment by RickHolc** [comment]
After installing Arduino 1.8.10 and reinstalling the ESPExceptionDecoder, now each time I start it I am asked to locate the *.Elf file. Before this never happened. What config file contains the path to the Elf file. Can I update it so it never asks again?

**Comment by nicechocolate** [comment]
Makes sense that recompiling is needed, once you think about what the decoder is trying to achieve and how it will do it.

I, too, hadn't realised that it would be the compiler that made the ELF that it needs.  Very helpful to know this.

**Comment by SHERAMII** [comment]
> 
> 
> ...Compile the sketch and it will not ask for elf...
> someone wrote a few days ago

thanks this worked
